CONTRIBUTING.md

```diff
@@ -43,24 +51,23 @@ Please set Following permissions for your bot:
     - Mention Everyone
     - Use slash commands
 
-After providing all the permissions you will get an url at the bottom of the page use that to invite the bot to your server.
+After providing all the permissions, a URL will be generated below.
 
-Open the Url you get and invite the bot to your test server.
+- Copy/paste this URL in a new browser window
+- Select your private server from the dropdown and authorize
+- This will invite the bot to your private discord server
 
 ## Setting Up Local Development
 
-- Clone the Repository to your machine
-- Now, get the .env file created above in the project folder
 - run `npm install`
-- Now, run the command `npm run register`
-
-This will register all the commands to your discord bot.
+- Now, run the command `npm run register` - this will register all the commands to your discord bot.
 
-Now let's link our local development server to our bot.
+Next you will have to set up the wranler cli, so that you can connect to your cloudflare workers account.
```

Comment:
Nitpik: it should be `wrangler cli` instead of `wranler cli` right?

Reply:
Hehe nice catch!